packages/docs/src/routes/qwikcity/content/component/index.mdx

```diff
@@ -45,7 +45,7 @@ src/
     └── some/
         └── path/
             ├── index.tsx              # https://example.com/some/path
-            └── sub_component.tsx
+            └── _sub_component.tsx
```

Comment:
Can you explain the addition of the underscore?

thanks

Reply:
line 58 there is an import from _sub_component that's why I changed it, you also change the import statement but I thought this would be more relevant.